tests/common.go

```diff
@@ -802,13 +812,10 @@ func ParseFlags() {
 				logrus.Infof("Backup driver found %v", backupDriver)
 			}
 		}
-		provider, _ := os.LookupEnv("VOLUME_PROVIDER")
```

Comment:
as we removed this where are we getting this info from?

Reply:
This env variable was introduced by https://github.com/portworx/torpedo/pull/462/files#diff-aae5bfbcffe751257212ace370c32505R798

we had volume driver portworx, but we needed to notify torpedo that it runs in other environment.It was needed because we used pxd as storage driver, but we needed to notify torpedo that it actually runs on aws or azure. Right now we pass correct info throught these variables, we don't need to duplicate it.

STORAGE_DRIVER
STORAGE_PROVISIONER